integration_tests/tests/forseti/controls/model.rb

```diff
@@ -0,0 +1,82 @@
+/**
+* Copyright 2018 Google LLC
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*      http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+require 'json'
+control 'model' do
+
+    describe "Model" do
+        before :context do
+            command("forseti inventory purge 0").result
+            command("forseti inventory create").result
+
+            # This variable cannot be used after all the inventories have been purged..
+            @inventory_id = JSON.parse(command("forseti inventory list").stdout).fetch("id")
+            command("forseti model create --inventory_index_id #{@inventory_id} model_new").result
+            command("forseti model use model_new").result
+        end
+
+        describe "Create and get a model" do
+
+            it "should be visible from the command-line" do
+                expect(command("forseti model get model_new").stderr).to eq ""
```

Comment:
Check on `stdout`

Reply:
Added it back.

Reply:
Seems it's still checking on `stderr`.

Reply:
If you have the `stdout`, then you really don't need the `stderr` check here.

Reply:
I thought it would be nice to check on both `stdout` and `stderr`. Removed `stderr` checks as requested.